data_dir/cdc_replication_profile.yaml

```diff
@@ -0,0 +1,29 @@
+keyspace: ks1
+table: table1
```

Comment:
Move it before table definition. otherwise c-s stress failed to create table

Reply:
Actually it does create the table successfully (tested). The order of config options should not matter, if C-S starts by loading everything into memory first (I guess it does).

I can reorder it anyway to be consistent with other profiles.